docs/content/help/contact_us.md

```diff
@@ -0,0 +1,16 @@
+# Contact Us
+
+## Mailing Lists
+
+The best way to contact the MOOSE team is to utilize the mailing list:
+
+* MOOSE Users for technical Q&A (moderate traffic): [moose-users@googlegroups.com](https://groups.google.com/forum/#!forum/moose-users)
+* MOOSE Announcements (very light traffic): [moose-announce@googlegroups.com](https://groups.google.com/forum/#!forum/moose-announce)
+
+!!! note "Gmail Users"
+    Just click the "Join group" button
+
+!!! note "Non-Gmail Users"
+    Send an email to:
+    * [moose-users+subscribe@googlegroups.com](mailto:moose-users+subscribe@googlegroups.com) and/or
+    * [moose-announce+subscribe@googlegroups.com](mailto:moose-announce+subscribe@googlegroups.com)
```

Comment:
I would like to change our admonition to the readthedoc theme style, I think we just need to copy paste some stuff from that theme into css/moose.css. Run with --theme readthedocs and you will see what I am after.